tests/e2e/local/vagrant/README.md

```diff
@@ -21,7 +21,10 @@ sh install_prereqs.sh
 
 # Steps
 ## 1. Set up Vagrant Environment
+The script will look for the istio source inside `~/go/src/istio.io`. If you have it in another place, set the `ISTIO` environment variable to point to the correct location.
+
 ```bash
+export ISTIO=`~/path/to/istio.io
```

Comment:
Is the \` here intentional or just a copy-paste extra from above?

Reply:
indeed.. apologies for my sloppiness